src/interfaces/chain.h

```diff
@@ -148,6 +148,11 @@ class Chain
     //! information.
     virtual bool findFirstBlockWithTimeAndHeight(int64_t min_time, int min_height, const FoundBlock& block={}) = 0;
 
+    //! Find next block if block is part of current chain. Also flag if
+    //! there was a reorg and the specified block hash is no longer in the
+    //! current chain, and optionally return block information.
+    virtual bool findNextBlock(const uint256& block_hash, int block_height, const FoundBlock& next={}, bool* reorg=nullptr) = 0;
```

Comment:
nit: parameter names in the function declaration differ from ones in the function definition:
- `FoundBlock& next` vs `FoundBlock& block_out`
- `bool* reorg` vs `bool* reorg_out`